glide.yaml

```diff
@@ -38,3 +38,12 @@ import:
   version: c55201b036063326c5b1b89ccfe45a184973d073
 - package: gopkg.in/yaml.v2
   version: a83829b6f1293c91addabc89d0571c246397bbf4
+
+testImports:
+- name: github.com/stretchr/testify
+  version: ^1.1.4
+  subpackages:
+  - assert
+  - require
+- package: github.com/leanovate/gopter
+  version: bc22a5c65d130b2f63f3572121ec03117d060039
```

Comment:
The pinned SHA is different from the SHA in the lock file?

Reply:
Hmm, I'll try running `glide udpdate` again.